test/com/google/enterprise/adaptor/database/DatabaseAdaptorTest.java

```diff
@@ -449,10 +465,20 @@ public void testSingleDocSqlResultSetHasOneRecord() throws SQLException {
             new GroupPrincipal("dgroup2"),
             new GroupPrincipal("dgroup1")))
         .build();
-    ResultSet rs = executeQuery("select * from data");
-    rs.next();
-    Acl acl = DatabaseAdaptor.buildAcl(rs, ",", DEFAULT_NAMESPACE,
-        ResultSetNext.STOP_NEXT);
+
+    Map<String, String> configEntries = new HashMap<String, String>();
+    configEntries.put("db.uniqueKey", "id:int");
+    configEntries.put("db.everyDocIdSql", "select id from data");
+    configEntries.put("db.singleDocContentSql",
+        "select * from data where id = ?");
+    configEntries.put("db.modeOfOperation", "rowToText");
+
+    DatabaseAdaptor adaptor = getObjectUnderTest(configEntries);
+    MockRequest request = new MockRequest(new DocId("1"));
+    RecordingResponse response = new RecordingResponse();
+    adaptor.getDocContent(request, response);
+
+    Acl acl = response.getAcl();
```

Comment:
It's more consistent now, but if we're planning to use this as an exemplar, I think it's nicer to just inline this value in the assert.

Reply:
Done.